stacks/spica/src/storage/components/storage-dialog-overview/storage-dialog-overwiev.spec.ts

```diff
@@ -1,41 +0,0 @@
-import {HttpClientTestingModule} from "@angular/common/http/testing";
```

Comment:
I'd expect new tests to be added.  
Can we add at least an "it block" that checks whether the data property is passed to the storage-view component?